wgpu-core/Cargo.toml

```diff
@@ -27,16 +27,17 @@ arrayvec = "0.5"
 bitflags = "1.0"
 copyless = "0.1"
 fxhash = "0.2"
+log = { version = "0.4" }
```

Comment:
is it intentionally not just `log = "0.4"`?

Reply:
Nah, just expected to need to turn off features, but didn't need to.